bot.py

```diff
@@ -127,7 +128,7 @@ async def add(ctx):
             }
         )
     )
-    add_to = await bot.wait_for("message", timeout=60 )
+    add_to = await bot.wait_for("message", timeout=60)
```

Comment:
This won't block the merge, but I wonder if a message in any channel will trigger this or if we should wrap this in a loop to only accept a message from the original author.